apps/web/src/project/components/delete-project-dialog.tsx

```
@@ -74,7 +81,7 @@ export function DeleteProjectDialog({
 					<Button variant="outline" onClick={() => onOpenChange(false)}>
 						Cancel
 					</Button>
-					<Button variant="destructive" onClick={onConfirm}>
+					<Button variant="destructive" onClick={onConfirm} disabled={confirmationInput != REQUIRED_CONFIRMATION_TEXT}>
```

Biome (2.4.13)

[error] Using != may be unsafe if you are relying on type coercion.

(lint/suspicious/noDoubleEquals)